spec/lib/guardian_spec.rb

```diff
@@ -4469,4 +4469,24 @@
       end
     end
   end
+
+  describe "#is_developer?" do
+    after { Developer.rebuild_cache }
+
+    it "returns true if user is an admin and has an associated `Developer` object" do
+      Developer.create!(user: admin)
+
+      expect(Guardian.new(admin).is_developer?).to eq(true)
+    end
+
+    it "returns false if user is an admin but does not have an associated `Developer` object" do
+      expect(Guardian.new(admin).is_developer?).to eq(false)
+    end
+
+    it "returns true if user's email has been configured as part of `Rails.configuration.developer_emails`" do
+      Rails.configuration.expects(:developer_emails).at_least_once.returns([user.email])
```

Comment:
Not saying you have to, but since you don't care about the expectation that the configuration receives this method, you could do this instead:

```suggestion
      Rails.configuration.stubs(:developer_emails).returns([user.email])
```